### ISSUE TITLE
Super grabbing feature

### ISSUE THREAD

**Issue by BannZay** [ddnet/ddnet#486]
Sometimes you need to move your part which contain game+front+tele layers, then you need to do 6 steps => grab - draw - grab - draw - grab - draw. Might be very annoing. Right?
So what about to make brush mode which grabs all layers informatiion and draw it.

**Comment by def-** [comment]
All layers would be wrong, some layers could have offsets. Then you'd have to select a group of layers, which complicates things. All in all, I think this would be an overcomplication.

**Comment by BannZay** [comment]
I gues its rather just "how it should work questions".  I see this as needed and some innovation feature.
Sadly about decline

**Comment by def-** [comment]
Well, I don't see an easier way since we can't be sure about where all kinds of layers are supposed to be positioned relative to that part. Only the mapper knows, which is why it's manual work.

**Comment by BannZay** [comment]
I guess just copy game+front+tele+switch+tune+speedup should be perfect

**Comment by Soreu** [comment]
And as far as I remember at this moment, for layer to have any offset it needs to be specified in options of its group
So maybe like... grab all visible layers (from whitin group) at once. And if you'd want to copy all but e.g. speedup, you would just hide the specific layer

**Comment by timakro** [comment]
Yes as Soreu said its possible in groups, they can't be offset. The hard thing is to find a way to use this since there is nothing similar where you edit multiple layers at once yet.

**Comment by 12pm** [comment]
Please try to implement it, Soreu's idea is good enough and would make mappers life a lot easier.
